### ISSUE TITLE
loss  source 3 in the paper of Be your own teacher

### ISSUE THREAD

**Issue by YANGWAGN** [ArchipLab-LinfengZhang/pytorch-self-distillation-final#1]
Thank you for your work and this code. However, I didn't find the implementation of the loss source 3  in the code. Thanks!

**Comment by ArchipLab-LinfengZhang** [comment]
Thanks for your attention on this work  : )
I have added the implementation of feature distillation loss in the latest version. 
Please refer to the line160-164 in train.py.
